packages/calcite-components/src/components/card/card.scss

```diff
@@ -1,87 +1,93 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-card-background-color: Specifies the background color of the component.
+ * @prop --calcite-card-border-color: Specifies the border color of the component.
+ * @prop --calcite-card-box-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
```

Comment:
I feel like the names of CSS variables should mirror the web-platform names, so instead of `corner-radius` here, it should be `border-radius`.  That doesn't mean that the source token name can't still be called `corner-radius` in the JSON because when that source token is transformed to an iOS or Android platform variable, it will be named specific to that platform.  Since we're using the CSS terms `box-shadow`, `background-color`, `border-color`, `color`, etc. in the name of a css property, I would expect the name `border-radius` to be used in the name of the variable that assigns to the `border-radius` css property.

Reply:
Will defer to @alisonailea on this one.

Reply:
I just think that makes sense since the target audience for these variables are web developers, so we should use terms they are familiar with.

Reply:
My 2 cents, I think we should follow the token names as it'll make it easier for documentation and maintenance. It also doesn't tie it to the implementation.

Reply:
@eriklharper while this particular instance of token names is web specific it would be a mistake to make naming decisions exclusively based on supporting a single platform given that we are already in discussions with teams building for multiple other platforms. Using the name `corner-radius`, it is still clear to everyone what we are discussing but does not limit token patterns to a web platform.